src/NHSD.BuyingCatalogue.Ordering.Api/NHSD.BuyingCatalogue.Ordering.Api.csproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.JwtBearer" Version="3.1.3" />
```

Comment:
Just need to be aware this is a newer version of the package compared with what's defined in the Organisations.Api project. No need to change it. If anything we should be updating all the packages within other projects. I think this will be a task for next sprint.

Reply:
👍